app/views/layouts/organisations.html.erb

```diff
@@ -0,0 +1,21 @@
+<% content_for :before_content do %>
+  <%= govuk_back_link(
+    text: 'Back',
+    href: :back,
+  ) %>
+<% end %>
+
+<% content_for :content do %>
+  <h1 class="govuk-heading-l">
+    Your Organisation
+  </h1>
+
+  <%= render TabNavigationComponent.new(items: [
+    { name: t('Details'), url: details_organisation_path(@organisation) },
+    { name: t('Users'), url: users_organisation_path(@organisation) },
+  ]) %>
+
```

Comment:
Can we add the following visually hidden title here. I’ve been meaning to add this to the prototype.

```suggestion

  <h2 class="govuk-visually-hidden"><%= PAGE_TITLE %></h2>
```

Where `PAGE_TITLE` is either ‘Details’ or ‘Users’